src/Middleware/SignatureHTMLSyntaxCheck.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Railroad\Railforums\Middleware;
+
+use DOMDocument;
+use Illuminate\Http\Request;
+use Closure;
+
+class SignatureHTMLSyntaxCheck
```

Comment:
Is this check even necessary?  I can't find anything that gets through the sanitization check that causes this to fail.

Reply:
Removed as the sanitization process removes invalid HTML as well